scripts/factory-hooks/db-update/db-update.sh

```diff
@@ -20,34 +20,44 @@ env="$2"
 # Database role. This is a truly unique identifier for an ACSF site and is e.g.
 # part of the public files path.
 db_role="$3"
-# Internal (Acquia managed) domain name of the website. (No public domain name
-# is assigned yet, immediately after installation.) The first part is a name
-# that is unique per installed site. A small but significant difference with
-# $db_role: if a site gets deleted and reinstalled with the same name, it gets
-# a different $db_role.
-internal_domain="$4"
-# To get only the site name in ${name[0]}:
-IFS='.' read -a name <<< $internal_domain
+# The public domain name of the website. If the site uses a path based domain,
+# the path is appended (without trailing slash), e.g. "domain.com/subpath".
+domain="$4"
+
+# BLT executable:
+blt="/mnt/www/html/$sitegroup.$env/vendor/acquia/blt/bin/blt"
```

Comment:
Duplicate now, because it also still exists below (just above the addition).

Also, the 'custom-arg' argument in the comment block above, is an argument to the db-update hook, not to the post-site-install hook (where it has now also appeared).

Also, if we are changing db-update.sh anyway, to the point the PR has grown now... I'd like to also change the comment just below saying "You need the URI of the site factory website in order for drush to target that site." I only just spotted that now.
* We are actually targeting the site using $domain nowadays, not using this $uri
* "$domain" is also the site URI, so the comment is ambiguous
* So there is a reason to use uri.php,  but different than what the comment says. (being AFAIK: the blt command needs the site name)

I'm going to rebase this PR on 11.x now, so I'll make those above two changes at the same time. Will comment about the rebased result in https://github.com/acquia/blt/pull/4061.